### PR TITLE
Add bare js fetch query

### PR DESCRIPTION
**Pull request by jzwang8**
This is untested--initial thoughts on how to connect with frontend.

### CODE REVIEW

File: frontend/src/match.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+function addParticipant() {
+  const params = new URLSearchParams();
+  fetch('/add-participant', {method: 'POST', body: params});
+}
+
+/** Search for match in list of all matches */
+function searchMatch() {
+  fetch('/search-match').then(response => response.json()).then((match) => {
+    // No match yet
+    if (match.id == 0) {
```
Comment (dhruvi618) [inline frontend/src/match.js:18]: What do you think about sending back a `matchStatus` which is true or false based on whether a match was found or not?

File: frontend/src/match.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      // Search every 5 seconds
+      setTimeout(searchMatch, 5000);
+    }
+    // Found match (eventually new page)
+    // @Front-end: ADD 'match-container' to CSS!
+    else {
+      const matchElement = document.getElementById('match-container');
+      matchLdapElement.innerText = match.firstldap + " " + match.secondLdap + " " + match.duration;
```
Comment (tcwanggoogle) [inline frontend/src/match.js:26]: nit: "l" and "L" in two variable names? Also, if renaming is not hard, I'd suggest we replace "ldap" with "username" for clarity.

**Comment by dhruvi618** [comment]
Can we wait to merge this until I finalize the corresponding UI portion and learn more about the details relating to fetching and posting using React? I currently don't have the `match-container` element and am still working on getting the states and properties of all the form inputs

**Comment by jzwang8** [comment]
Not pushing these changes to master

**Pull request closed without merging**